### ISSUE TITLE
Support running tests via `elixir`

### ISSUE THREAD

**Issue by G-Rath** [KronicDeth/intellij-elixir#1399]
I'm doing Elixir exercises on Exercism, which have both the exercise file & test file in the root of the project:

```
 .
 | - bob.exs
 | - bob_test.exs
```

The project doesn't make use of `mix`, so I can't run `mix test`.

Instead, the instructions are to just run `elixir bob_test.exs`.

It'd be good if test-related options were provided for files with `_test`, regardless of their directory.

**Comment by G-Rath** [comment]
I'm new to Elixir, so I'm possibly making a fool of myself here, by suggesting something that is not recommended.

My main point of this issue is: Hey, Exercism lay out their exercises in this format, which works - It'd be nice if `intellij-elixir` handled itself nicely the same way Ruby, JS, TS, etc do (such as letting us run tests without a `test` directory, directly from a single file).

Let me know if there's anything I can provide to help with this :)

**Comment by KronicDeth** [comment]
If you need to run `elixir bob_test.exs`, you can use the Elixir Run Configuration for now.  Having the gutter run icons is #688.

**Comment by G-Rath** [comment]
Does that give you the nice test results view?

**Comment by KronicDeth** [comment]
No, you're right, using just `elixir` it won't know to attach the graphical formatter.  The way the graphical test output works is that it is setting the ExUnit formatter to one that can output the TeamCity format used by JetBrains IDEs.  I'll need to add custom support to do that without `mix test`.

**Comment by G-Rath** [comment]
Awesome! Is that somethings that's feasible?

How much work would be involved in adding custom support?

**Comment by KronicDeth** [comment]
For new run configurations you just copy the one that is closest and modify it to fit the options you need.  It involves like 1-3 classes.  You also need to come up with a separate icon if you want to be fancy.

**Comment by G-Rath** [comment]
Just for clarity: That's for adding the feature to this plugin right?

I've not done Java in a long while, and never worked with IntelliJ plugins, but I'm happy to give it a go if you don't think it's too complex :)
